### ISSUE TITLE
Update to current Touchdesigner API planned?

### ISSUE THREAD

**Issue by auenkind** [DBraun/PyTorchTOP#17]
Are there any plans to Update the plugin to work with the current Touchdesigner version? 

I tried to do the API update myself but i`m stuck at the point where the foreground CudaArray is copied to myInputTensorForeground whatever i tried it results in a cudaErrorInvalidValue.

My Setup is:
- Touchdesigner 2022.31030
- CUDA 11.2

As is see a lot of usecases for this in the future i would sponsor the update commercially.

**Comment by DBraun** [comment]
Yes, but there's an issue with libtorch versions above 1.7.1 and TouchDesigner.

I'm subscribed to these issues:
https://github.com/pytorch/pytorch/pull/67632
https://github.com/pytorch/pytorch/issues/61111

Once they're resolved I'll try updating PyTorch TOP. Until then I recommend sticking with this exact PyTorch version https://download.pytorch.org/libtorch/cu110/libtorch-shared-with-deps-1.7.1%2Bcu110.zip Maybe TouchDesigner 31030 will work but I haven't tried.

**Comment by auenkind** [comment]
Thanks for this Info. I also experienced similar crashes with Version 2022.31030 but was not sure if they originated from my code. I will switch to do background matting in an external application and use NDI to feed the result to Touchdesigner.

**Comment by DBraun** [comment]
I also recommend https://docs.derivative.ca/Nvidia_Background_TOP

**Comment by auenkind** [comment]
Thanks, unfortunately that does not work for my use-case as i need full body tracking. My goal is to integrate https://github.com/PeterL1n/RobustVideoMatting
